packages/react-components/source/react/library/form/internal/FormActions.js

```diff
@@ -58,7 +61,7 @@ const FormActions = ({
       className="rc-form-action"
       buttonType="submit"
       loading={submitting}
-      disabled={disabled || !isValid}
+      disabled={disabled || !isValid || submitDisabled}
```

Comment:
Is it possible to achieve this same functionality by using the existing isValid flag? Looks like we could use either the required or validator function props for individual fields which will set isValid.

Reply:
We can't set `isValid` that is set when the form is invalid - https://github.com/puppetlabs/design-system/blob/3d9ea221b614ac70313f2200341920c549104647/packages/react-components/source/react/library/form/internal/methods.js#L53

This will allow us to control the ability of the submit, the way we are going to use it we can't validate the form as we are going to hide some of it until the user selects the first option in the form.